packages/lpa-questionnaire-web-app/src/config.js

```diff
@@ -0,0 +1,59 @@
+// const oneGigabyte = 1024 * 1024 * 1024;
+
+module.exports = {
+  appeals: {
+    timeout: Number(process.env.APPEALS_SERVICE_API_TIMEOUT || 10000),
+    url: process.env.APPEALS_SERVICE_API_URL,
+  },
+  db: {
+    session: {
+      uri: process.env.SESSION_MONGODB_URL,
+      databaseName: process.env.SESSION_MONGODB_DB_NAME,
+      collection: process.env.SESSION_MONGODB_COLLECTION || 'sessions',
+      expiresColumn: '_ts',
+      expires: 1000 * 60 * 60 * 24 * 14, // value in milliseconds
+      expiresAfterSeconds: 60 * 60 * 24 * 14, // value in seconds
+      connectionOptions: {
+        useNewUrlParser: true,
+        useUnifiedTopology: true,
+      },
+    },
+  },
+  documents: {
+    timeout: Number(process.env.DOCUMENTS_SERVICE_API_TIMEOUT || 10000),
+    url: process.env.DOCUMENTS_SERVICE_API_URL,
+  },
+  fileUpload: {
```

Comment:
I suspect we won't need the `fileUpload` object (unless we're uploading files from this questionnaire)

Reply:
We will be uploading files for this, hence why I have left this in